components/director/hack/config-local.yaml

```diff
@@ -168,6 +168,7 @@ clientCredentialsRegistrationScopes:
     - "webhook:write"
   application:
     - "application:read"
+    - "applicationByLocalTenantIDAndAppTemplateID:read"
```

Comment:
You are not introducing new scope. You should add "applicationByLocalTenantIDAndAppTemplateID: ["application:read"]" in the query section